### DISCUSSION TITLE
Doesnt Work with Non UTF Windows accounts

### DISCUSSION THREAD

**Discussion by neosinan** [pyinstaller/pyinstaller#5571] [Help] [answered]
My Windows account name has Non UTF 8 charter on I get this error,  Maybe It Doesnt Work with Non UTF Windows accounts

File "C:\Users\Sinan Çağrı Kurt\AppData\Local\Programs\Python\Python37-32\Scripts\pyinstaller-script.py", line 1
SyntaxError: Non-UTF-8 code starting with '\xe7' in file C:\Users\Sinan Çağrı Kurt\AppData\Local\Programs\Python\Python37-32\Scripts\pyinstaller-script.py on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

**Comment by bwoodsend** [comment, marked as answer]
That error isn't complaining directly about a path. It's saying the `pyinstaller-script.py` itself contains a non UTF-8 character but doesn't have the `# -*- coding: utf-8 -*-` shebang at the top. If you open `C:\Users\Sinan Çağrı Kurt\AppData\Local\Programs\Python\Python37-32\Scripts\pyinstaller-script.py` I'm guessing it has a path to the Python interpreter in a comment at the top which will contain your username? In which case this is a bug.
